demo/MultiValueListControl/src/buildInteractionModel.ts

```diff
@@ -31,41 +34,41 @@ export namespace MultiValueListDemoIM {
         .addOrMergeSlotType(yesNoMaybeSlotType)
         .addOrMergeSlotType(filteredYesNoMaybeSlotType)
         .addOrMergeSlotType({
-            name: 'AppleSuite',
+            name: 'GroceryItem',
             values: [
                 {
-                    id: 'AirPods',
+                    id: 'Milk',
                     name: {
-                        value: 'AirPods',
-                        synonyms: ['pods', 'earphones', 'ipod'],
+                        value: 'Milk',
+                        synonyms: ['milk', 'almond milk', 'low fat milk'],
                     },
                 },
                 {
-                    id: 'iWatch',
+                    id: 'Eggs',
                     name: {
-                        value: 'iWatch',
-                        synonyms: ['watch', 'wrist wear'],
+                        value: 'Eggs',
+                        synonyms: ['egg', 'eggs'],
                     },
                 },
                 {
-                    id: 'iPhone',
+                    id: 'Ceral',
```

Comment:
nit: Ceral -> Cereal

Reply:
I've had fixed this is recent commit, maybe you didn't look at latest file